mfr/extensions/jsc3d/static/js/jsc3d.js

```diff
@@ -5506,7 +5506,7 @@ JSC3D.StlLoader.prototype.loadFromUrl = function(urlName) {
 	//TODO: current blob implementation seems do not work correctly on IE10. Repair it or turn to an arraybuffer implementation.
 	var isIE10Compatible = false;//(isIE && parseInt(JSC3D.PlatformInfo.version) >= 10);
 	var xhr = new XMLHttpRequest;
-	xhr.open('GET', encodeURI(urlName), true);
+	xhr.open('GET', urlName, true);

```

Comment:
Add a comment here for the change of using `furl` which already encoded the `urlName`.